plugins/modules/ome_configuration_compliance_baseline.py

```diff
@@ -93,6 +93,21 @@
       - Description of the compliance baseline.
       - This option is applicable when I(command) is C(create), or C(modify).
     type: str
+  run_later:
+    description:
+      - Indicates whether template is to be assigned immediately or in the future.
+      - If I(run_later) is C(true), then I(staged_at_reboot) is ignored.
+      - If I(run_later) is C(true), then I(job_wait) is not applicable.
+    type: bool
+  cron:
+    description:
+      - Provide a cron expression based on Quartz cron format.
```

Comment:
Describe the format - and also mention it is a specific datetime

Reply:
done